Makefile

```diff
@@ -118,7 +118,11 @@ docker-test-unit: $(DOCKER_CACHE)
 .PHONY: docker-test-integration
 docker-test-integration: .coverage
 	@echo "===> Building Antrea Integration Test Docker image <==="
-	@docker build --pull -t antrea/test -f build/images/test/Dockerfile .
+	@if [[ $DOCKER_REGISTRY != "" ]]; then \
```

Comment:
@lzhecheng I think you should use `ifneq` for portability (does `if` work on Windows?)

```make
docker-test-integration: .coverage
ifneq ($(DOCKER_REGISTRY),"")
        docker build -t antrea/test -f build/images/test/Dockerfile .
else
        docker build --pull -t antrea/test -f build/images/test/Dockerfile .
endif
```

Reply:
Indeed. Thank you!